src/main/java/fi/helsinki/cs/tmc/cli/command/SubmitCommand.java

```diff
@@ -30,9 +32,13 @@ public void run(String[] args) {
         TmcCore core;
         Course course;
         SubmissionResult submit;
+        DirectoryUtil dirUtil;
 
+        dirUtil = new DirectoryUtil();
         core = this.app.getTmcCore();
-        course = TmcUtil.findCourse(core, "cert-test");
+        Path dir = dirUtil.getCourseDirectory();
+
```

Comment:
Here should be check if the dir is null and tell the user that the exercise doesn't exist.